### ISSUE TITLE
Issue with splitting chapters

### ISSUE THREAD

**Issue by PolarisFx** [dipu-bd/lightnovel-crawler#1234]
I've had an issue with multiple versions now where if you choose to split the files, the splits don't happen where you expect them to. And you end up with random chapters in different files, and sometimes the files themselves don't contain the chapters in the filename.

Case in point, my latest crawl off boxnovel.org: https://imgur.com/a/BlFhIqR

In this file I would expect to find Chapters 1-100, however I find Chapters 300-349 & then Chapters 629-678. I ended up finding Chapters 1-10 in "Trash of the Count s Family Chapter 301-400" sandwiched in between chapters 718 & 61: https://imgur.com/a/SLxaEdc

Downloading novels as a single file has always worked flawlessly. 

**App source**:    PIP
**App version**:   2.28.11
**Your OS**:       Linux

**Comment by dipu-bd** [comment]
Generally it shold serialize chapters in order of appearance. In your case, it might be another problem. Please give me a novel link where this issue happens so that I can test and fix it

**Comment by PolarisFx** [comment]
I should've updated this, but it slipped my mind.

Pretty sure at this point it was my choice of source. After this issue arose I stopped splitting files and still I had chapters out of order on multiple novels. I thought maybe it was MoonReader or Calibre showing the chapters out of order so I tried multiple apps and the issue persisted, eventually I started checking out the sites where the crawler was pulling from and found the problem.

This is the source site from my issue above: https://boxnovel.org/novel/trash-of-the-counts-family

lncrawl is doing its job flawlessly, it does indeed serialize in the order of appearance however certain sites don't feel the need to display the chapters in the proper order. I was having trouble finding good sources to use and I had a streak of good luck with boxnovel so I stuck with them, turns out multiple novels are out of order, I've since found a more reliable source and haven't had any issues since.
